lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -172,6 +208,11 @@ defmodule Kernel.LexicalTracker do
   defp add_reference(d, module, :compile) when is_atom(module),
     do: :ets.insert(d, {{:mode, module}, :compile})
 
+  defp add_reference(d, module, fas, line, :runtime) when is_atom(module),
+    do: :ets.insert_new(d, {{:mode, module, fas, line}, :runtime})
```

Comment:
Why would we want to track multiple references to the same MFA on the same line? :)